_episodes/interviews/2024-12-19-daniel-thompson-yvetot.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Tauri 2.0 with Daniel Thompson-Yvetot"
+date: 2024-12-19T17:00:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2024-12-19-daniel-thompson-yvetot.mp3
+duration: "44:28"
+length: "64279919"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Daniel Thompson-Yvetot](https://github.com/nothingismagick), co-founder of [Tauri](https://tauri.studio/) about Tauri's 2.0 release. Tauri is a toolkit that helps developers make applications for the major desktop platforms using a variety of front-end frameworks.
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps 
+- [@00:33] - Who is Daniel and what is Tauri?
+- [@10:12] - Small app size and consequent app security/vulnerability issues
```

Comment:
This isn't super descriptive — can you come up with a more concise segment title, or split it into two?